OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -286,10 +286,10 @@ public IModInputCombination RegisterCombination(IModBehaviour mod, string name,
                 var collisions = GetCollisions(combo.Hashes);
                 foreach (string collision in collisions)
                 {
-                    _console.WriteLine(collision);
+                    _console.WriteLine("\"" + collision + "\"");
```

Comment:
_logger?
$"\"{collision}\"");

Reply:
that's part of the error, so not _logger

Reply:
Ok, then it's probably nice with some description in the error message, such as: "Combination already exists: ..."

Reply:
do you even read? (look 3 lines higher)

Reply:
It would be better to `String.Join` all the collision strings and put it all in a single `WriteLine`, if possible.

So, if a user tries to use the same combo in two different mods, they get a console error, and nothing else happens, right? Just thinking because most users won't see the logs. With the current implementation, is it too difficult to allow one combo to be used by multiple mods?

Reply:
I'm working on adding a proper and somewhat user-friendly combination editing tool which should report collsions in-game